frontend/.storybook/preview.tsx

```diff
@@ -5,6 +5,7 @@ import { HelmetProvider } from 'react-helmet-async'
 import { QueryClient, QueryClientProvider } from 'react-query'
```

Comment:
would be helpful if we can add a comment in this file saying that it's used for global storybook config, and link to the docs. I know the docs are explicit about this but helps to save a few seconds/minutes of confusion for future devs since this file isn't imported anywhere

Reply:
921b87c